wallet/lib/pages/purchase_item/purchase_item_screen.dart

```diff
@@ -684,7 +669,32 @@ class _OwnerBottomDrawerState extends State<OwnerBottomDrawer> {
               ),
             ),
           ),
-        )
+        ),
+        Align(
```

Comment:
Add a test for this.

Reply:
Its test case is already added. I did not add this widget I changed its position in the widget tree.


<img width="881" alt="image" src="https://user-images.githubusercontent.com/115459890/204493190-9843ed18-0bbb-43f7-afce-f078ca84bb37.png">


<img width="930" alt="image" src="https://user-images.githubusercontent.com/115459890/204493050-e62d213d-1fe4-4c2b-89d1-342ddefdcf66.png">

Reply:
Then modify it so that it can also test its tap behavior.